modules/store/testing/src/testing.ts

```diff
@@ -3,20 +3,20 @@ import {
   FactoryProvider,
   Injector,
   ValueProvider,
-} from '@angular/core';
-import { MockState } from './mock_state';
+} from "@angular/core";
```

Comment:
Would it be possible to undo the quote changes (`'` vs `"`") please?

Reply:
Yeah, my bad. VSCode must have done a format-on-save when I did the refactor 🤦‍♂️

Fixed and amended commit.